audmetric/core/api.py

```diff
@@ -237,6 +341,118 @@ def edit_distance(
     return m1[len(truth)]
 
 
+def equal_error_rate(
+    truth: typing.Union[
+        typing.Union[bool, int, float],
```

Comment:
Here `float` should be removed from the type hint since `float` would raise an error in `detection_error_tradeoff`

Reply:
```suggestion
        typing.Union[bool, int],
```